ui/app/components/bn-as-decimal-input.js

```diff
@@ -128,15 +130,15 @@ BnAsDecimalInput.prototype.updateValidity = function (event) {
 }
 
 BnAsDecimalInput.prototype.constructWarning = function () {
-  const { name, min, max } = this.props
+  const { name, min, max, scale } = this.props
   let message = name ? name + ' ' : ''
 
   if (min && max) {
-    message += `must be greater than or equal to  ${min} and less than or equal to ${max}.`
+    message += `must be greater than or equal to  ${this.downsize(min.toString(10), scale)} and less than or equal to ${this.downsize(max.toString(10), scale)}.`
```

Comment:
can we use the `newMin` and `newMax` here?

Reply:
We can, but I just did it here to continue to draw from props only rather other sources.

Reply:
right, ok maybe just move the calculation to the top of the fn
or dont, w/e

Reply:
:+1: